include/constants/battle/condition.h

```diff
@@ -103,28 +103,26 @@
 #define FIELD_CONDITION_GRAVITY_SHIFT       12
 #define FIELD_CONDITION_TRICK_ROOM_SHIFT    16
 
-enum {
-    OVERWORLD_WEATHER_CLEAR = 0,
-    OVERWORLD_WEATHER_CLOUDY,
-    OVERWORLD_WEATHER_RAINING,
-    OVERWORLD_WEATHER_HEAVY_RAIN,
-    OVERWORLD_WEATHER_THUNDERSTORM,
-    OVERWORLD_WEATHER_SNOWING,
-    OVERWORLD_WEATHER_HEAVY_SNOW,
-    OVERWORLD_WEATHER_BLIZZARD,
-    OVERWORLD_WEATHER_CLEAR_8,
-    OVERWORLD_WEATHER_SLOW_ASHFALL,
-    OVERWORLD_WEATHER_SANDSTORM,
-    OVERWORLD_WEATHER_HAILING,
-    OVERWORLD_WEATHER_SPIRITS,
-    OVERWORLD_WEATHER_CLEAR_13,
-    OVERWORLD_WEATHER_FOG,
-    OVERWORLD_WEATHER_DEEP_FOG,
-    OVERWORLD_WEATHER_DARK_FLASH,
-    
-    // these are only for the Battle Frontier
-    OVERWORLD_WEATHER_HARSH_SUN = 1001,
-    OVERWORLD_WEATHER_TRICK_ROOM,
-};
```

Comment:
I moved these to `#define` syntax, since there is a break in values outside the usual step-increments.